src/View3D.ts

```diff
@@ -235,6 +246,11 @@ class View3D extends Component<View3DEvents> implements OptionGetters<Omit<View3
    * @readonly
    */
   public get ar() { return this._arManager; }
+  /**
+   * {@link AnnotationManager} instance of the View3D
+   * @type {AnnotationManager}
+   */
+  public get annotation() { return this._annotationManager; }
```

Comment:
Because it has a class called Annotation, the annotation seems to be confused. How about using annotationManager?